packages/core/parcel-bundler/src/assets/MarkdownAsset.js

```diff
@@ -0,0 +1,11 @@
+const localRequire = require('../utils/localRequire');
+const HTMLAsset = require('./HTMLAsset');
+
+class MarkdownAsset extends HTMLAsset {
```

Comment:
I wouldn't recommend extending from another asset type. Instead, we can rely on the pipeline to pass through the generated HTML to the HTMLAsset.

Extend from `Asset` here, and return the html from the `generate` method. For an example, see how e.g. JSONAsset does it.

Reply:
Rather the PugAsset:
https://github.com/parcel-bundler/parcel/blob/d7dc51f9eacd80aadd1e41888771033e447758f1/packages/core/parcel-bundler/src/assets/PugAsset.js#L8

https://github.com/parcel-bundler/parcel/blob/d7dc51f9eacd80aadd1e41888771033e447758f1/packages/core/parcel-bundler/src/assets/PugAsset.js#L36